admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/presentation/component/AdminConfirmationDialog.kt

```diff
@@ -0,0 +1,146 @@
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.unit.dp
+import net.thechance.mena.admin_panel.resources.Res
+import net.thechance.mena.admin_panel.resources.cancel
+import net.thechance.mena.designsystem.presentation.component.button.OutlinedButton
+import net.thechance.mena.designsystem.presentation.component.button.PrimaryButton
+import net.thechance.mena.designsystem.presentation.component.dialog.BasicDialog
+import net.thechance.mena.designsystem.presentation.component.icon.Icon
+import net.thechance.mena.designsystem.presentation.component.scaffold.ScaffoldScope
+import net.thechance.mena.designsystem.presentation.component.text.Text
+import net.thechance.mena.designsystem.presentation.theme.theme.Theme
+import org.jetbrains.compose.resources.stringResource
+
+@Composable
+fun ScaffoldScope.AdminConfirmationDialog(
+    dialogIcon: Painter,
+    confirmationIcon: Painter,
+    title: String,
+    description: String,
+    confirmationButtonText: String,
+    isVisible: Boolean,
+    onDismiss: () -> Unit,
+    onConfirm: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    BasicDialog(
+        onDismiss = onDismiss,
+        isVisible = isVisible,
+        dialogCornerShape = RoundedCornerShape(24.dp),
+        contentPadding = PaddingValues(24.dp),
+        hasDismissButton = false,
+        contentColor = Theme.colorScheme.background.surfaceLow,
+        actionButtons = {
+            AdminConfirmationDialogActionButtons(
+                onDismiss = onDismiss,
+                onConfirm = onConfirm,
+                confirmationIcon = confirmationIcon,
+                confirmationButtonText = confirmationButtonText
+            )
+        },
+        modifier = modifier.width(400.dp)
+    ) {
+        AdminConfirmationDialogContent(
+            dialogIcon = dialogIcon,
+            title = title,
+            description = description
+        )
+    }
+}
+
+
+@Composable
+private fun AdminConfirmationDialogActionButtons(
+    onDismiss: () -> Unit,
+    onConfirm: () -> Unit,
+    confirmationIcon: Painter,
+    confirmationButtonText: String
+) {
+    Row(
+        modifier = Modifier
+            .padding(top = 12.dp)
+            .fillMaxWidth(),
+        horizontalArrangement = Arrangement.spacedBy(8.dp)
+    ) {
+        PrimaryButton(
+            text = stringResource(Res.string.cancel),
+            onClick = onDismiss,
+            contentPadding = PaddingValues(
+                start = 16.dp,
+                end = 16.dp,
+                top = 8.dp,
+                bottom = 8.dp
+            )
```

Comment:
Check the design again
```suggestion
            contentPadding = PaddingValues(horizontal = 16.dp, vertical = 13.dp
            )
```

Reply:
they are correctly like design 16 and 8 not 13

Reply:
How? 
<img width="617" height="408" alt="image" src="https://github.com/user-attachments/assets/5a7c83ac-e7fd-4a21-ab23-790559658dae" />